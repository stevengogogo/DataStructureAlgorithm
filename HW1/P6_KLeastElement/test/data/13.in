200 500
-6132 5516 -5105 -9622 9187 5265 -9104 3937 -1640 5040 6902 -1940 8236 888 7291 -4613 -438 -7829 1206 5891 -9953 -2445 9325 -6999 -9810 6002 -2001 2841 -8599 5389 2458 -7197 -748 1096 -8694 -9603 7105 -7697 8815 6768 -6994 -6786 8470 -235 8964 6186 -7379 -328 3450 -8624 -5980 499 -9140 2959 -2818 8337 -5333 -4814 7354 -1238 1930 -2986 8649 4204 -2098 678 -8654 -1540 6703 2609 -960 1372 -4133 -5247 -3230 -8328 -9020 368 -5411 9340 8715 -9927 -8806 9564 8148 3296 6529 -5829 5660 9548 6800 -1223 -7237 -7097 161 -1576 -7746 6053 -9333 -3760 46 -1135 -3921 -2098 916 -9363 7827 -3496 -8150 9732 2664 8767 -5329 973 3926 -493 -3160 933 3721 -8710 -8651 5292 3107 9604 -5459 -2574 -6083 -1487 6596 8274 -8275 -4893 9518 -5867 -8649 6319 -2581 7858 -1356 6623 -3812 5319 165 -9731 -4626 5701 -1463 -4849 -8503 -2278 -9245 3943 7377 7496 -5526 -1680 -2939 -7070 4057 3120 -7357 8667 -9273 5012 -1179 7799 -388 8551 854 5774 -3846 -8506 4042 -8186 3508 -6768 6120 -8223 -2383 -1759 -6008 -9166 -2599 9894 8325 7801 -1679 6449 -1047 -4445 9464 4980 -6818 -6478 -8706 -2508 -5024 4426 -2712 1728
Insert 64 -7161
Query 117 118 1
Delete 100
Reverse 87 151
Query 68 139 13
Query 126 185 2
Query 97 187 42
Insert 192 503
Delete 72
Query 43 167 2
Query 170 193 9
Delete 120
Delete 183
Reverse 187 195
Insert 105 -6715
Insert 181 -7886
Query 199 200 2
Reverse 94 185
Reverse 159 175
Query 32 59 14
Query 18 40 23
Reverse 182 198
Insert 117 4359
Insert 149 -9828
Delete 96
Delete 53
Query 36 90 24
Delete 147
Insert 178 3922
Delete 44
Query 52 88 6
Reverse 71 115
Reverse 179 194
Query 33 73 5
Query 51 174 71
Query 170 190 12
Delete 165
Delete 104
Insert 52 2262
Delete 153
Query 4 51 29
Delete 181
Insert 22 -2021
Insert 132 934
Insert 189 -3914
Delete 88
Reverse 113 195
Delete 56
Delete 54
Insert 52 6338
Reverse 83 181
Query 98 149 8
Delete 174
Insert 185 -3704
Reverse 104 177
Reverse 83 154
Delete 191
Reverse 58 140
Delete 73
Delete 119
Delete 38
Insert 98 1954
Delete 128
Delete 125
Reverse 22 150
Query 24 105 79
Insert 54 6447
Query 150 164 6
Insert 172 -8661
Reverse 127 178
Insert 195 -9083
Delete 122
Reverse 104 158
Query 157 180 24
Insert 5 1340
Query 67 86 15
Reverse 32 183
Delete 158
Insert 5 -2078
Query 52 137 30
Query 111 190 36
Reverse 11 36
Reverse 92 161
Query 117 124 7
Query 16 118 97
Insert 161 -6138
Delete 149
Insert 40 -7926
Reverse 10 38
Reverse 85 171
Query 134 139 1
Query 62 95 10
Insert 187 4232
Query 167 190 10
Query 140 179 31
Reverse 130 176
Query 31 167 133
Insert 58 839
Query 177 178 2
Insert 184 -5542
Query 179 197 11
Insert 89 -7971
Delete 58
Insert 162 5258
Insert 17 -4846
Delete 34
Delete 118
Delete 42
Insert 111 -9730
Delete 77
Insert 84 5227
Insert 55 4338
Insert 122 -5112
Reverse 6 51
Delete 21
Query 38 95 30
Delete 100
Query 21 138 49
Insert 20 7283
Delete 60
Query 79 193 25
Reverse 142 184
Insert 160 8647
Insert 120 -1702
Insert 4 -7999
Reverse 34 90
Query 110 124 15
Delete 37
Delete 111
Query 190 191 1
Insert 35 -3211
Reverse 178 199
Delete 112
Delete 53
Delete 66
Reverse 190 192
Reverse 58 192
Reverse 67 174
Insert 21 7374
Reverse 55 147
Insert 18 7127
Query 80 154 44
Reverse 173 199
Query 45 124 30
Reverse 129 146
Delete 153
Delete 13
Reverse 131 157
Reverse 175 186
Query 170 178 5
Query 24 136 11
Insert 37 -9385
Reverse 103 144
Query 147 152 4
Reverse 95 125
Query 130 194 46
Delete 18
Query 54 118 15
Reverse 41 164
Insert 73 -6967
Reverse 6 41
Reverse 121 127
Insert 138 7531
Insert 22 5536
Insert 168 649
Delete 129
Insert 65 -6123
Delete 113
Query 86 122 33
Reverse 13 194
Reverse 33 102
Delete 180
Delete 106
Delete 188
Reverse 82 86
Delete 144
Insert 100 -3646
Query 193 194 2
Delete 144
Delete 149
Query 51 84 31
Reverse 21 51
Delete 90
Query 26 140 33
Insert 137 9211
Query 35 105 37
Reverse 175 184
Query 154 182 8
Query 27 41 10
Query 74 148 69
Reverse 39 88
Delete 73
Delete 161
Insert 190 -3040
Reverse 194 195
Insert 196 531
Delete 28
Query 141 166 5
Delete 153
Delete 188
Query 148 160 5
Query 177 180 1
Query 148 161 9
Delete 66
Insert 80 -8759
Query 57 134 68
Delete 120
Delete 107
Query 65 114 48
Reverse 36 124
Reverse 27 124
Query 12 121 38
Query 149 171 12
Insert 177 -3388
Reverse 105 170
Delete 114
Query 51 179 62
Delete 79
Delete 8
Insert 68 3430
Reverse 180 190
Query 150 185 27
Reverse 141 142
Query 78 120 16
Insert 175 6297
Delete 122
Insert 190 -6855
Reverse 68 140
Insert 96 1961
Insert 94 -1968
Reverse 17 108
Delete 13
Insert 45 -7501
Delete 151
Delete 137
Insert 182 -3287
Delete 116
Query 182 184 3
Query 124 145 1
Reverse 116 183
Reverse 52 84
Reverse 91 182
Delete 22
Reverse 41 68
Delete 70
Insert 177 -2797
Delete 13
Query 178 178 1
Reverse 113 165
Delete 126
Delete 183
Delete 114
Delete 93
Query 38 112 3
Insert 183 7246
Query 77 96 6
Insert 80 5631
Insert 63 -4804
Reverse 134 148
Query 85 134 19
Delete 100
Query 1 44 36
Delete 135
Insert 53 -2628
Query 54 109 47
Insert 5 8396
Delete 177
Query 119 172 30
Query 119 186 6
Insert 148 3358
Insert 8 -8504
Delete 113
Query 176 181 6
Reverse 97 151
Insert 104 -6826
Query 29 69 6
Delete 37
Delete 107
Delete 100
Delete 178
Delete 7
Insert 29 -5916
Delete 89
Delete 25
Reverse 16 124
Insert 128 -7900
Delete 28
Reverse 45 83
Reverse 106 183
Query 66 123 17
Reverse 20 120
Delete 11
Insert 43 -1003
Reverse 126 155
Reverse 136 148
Delete 92
Delete 68
Insert 65 1944
Query 66 108 1
Query 162 180 9
Query 169 182 5
Delete 16
Reverse 108 175
Insert 68 -5546
Reverse 129 163
Insert 170 2740
Query 12 50 3
Query 98 154 28
Query 17 65 37
Delete 94
Delete 8
Delete 130
Reverse 175 179
Delete 86
Reverse 32 166
Insert 156 7332
Delete 80
Delete 21
Query 63 108 43
Insert 119 6796
Query 11 130 66
Reverse 82 133
Insert 61 8026
Query 8 127 30
Reverse 150 176
Reverse 163 171
Reverse 6 103
Query 48 132 31
Reverse 18 41
Delete 35
Delete 88
Query 141 144 3
Query 36 85 49
Insert 6 7332
Insert 113 2020
Delete 61
Reverse 30 140
Query 80 151 69
Query 24 109 15
Query 55 173 106
Insert 76 325
Delete 23
Query 47 139 15
Reverse 87 121
Insert 117 194
Insert 160 -2880
Delete 22
Query 79 85 5
Reverse 116 162
Query 179 180 1
Delete 11
Query 93 114 9
Insert 130 -777
Query 123 161 8
Reverse 32 52
Delete 177
Insert 17 5464
Delete 31
Delete 107
Query 31 106 42
Query 49 83 35
Reverse 134 167
Delete 130
Delete 90
Delete 66
Query 92 135 7
Reverse 93 140
Reverse 170 174
Delete 13
Insert 124 -667
Query 169 171 1
Reverse 4 173
Delete 165
Query 171 172 1
Query 3 55 14
Insert 143 5561
Query 73 85 6
Reverse 28 106
Delete 167
Delete 123
Reverse 27 118
Insert 52 2034
Reverse 101 165
Insert 145 -7430
Delete 137
Query 63 173 15
Query 74 154 8
Insert 41 973
Query 24 24 1
Insert 54 7541
Reverse 93 108
Delete 24
Delete 174
Insert 156 2176
Delete 67
Reverse 3 113
Insert 63 -5722
Query 90 118 29
Reverse 24 67
Query 36 125 7
Delete 10
Delete 151
Insert 139 630
Query 168 168 1
Delete 121
Delete 62
Delete 158
Insert 154 4218
Query 27 89 23
Insert 143 2691
Delete 149
Reverse 116 125
Query 140 172 19
Reverse 5 153
Reverse 109 172
Reverse 88 149
Insert 99 4897
Delete 15
Reverse 129 166
Query 109 164 44
Reverse 130 160
Reverse 58 85
Query 11 89 76
Insert 9 8987
Query 146 154 1
Insert 18 -9330
Insert 153 6525
Insert 33 -6313
Delete 174
Reverse 76 98
Delete 25
Insert 138 -4868
Reverse 16 138
Reverse 127 136
Insert 162 3620
Delete 107
Delete 134
Delete 155
Delete 109
Insert 164 -2686
Query 23 107 59
Insert 34 1649
Delete 18
Reverse 22 45
Insert 37 -9571
Insert 79 7501
Reverse 26 149
Reverse 82 98
Delete 120
Insert 83 7128
Reverse 146 155
Delete 57
Reverse 83 172
Query 53 84 28
Reverse 46 130
Insert 99 2857
Query 91 173 26
Reverse 113 137
Reverse 27 135
Insert 138 8460
Query 41 47 4
Insert 81 -6400
Query 46 95 48
Reverse 6 152
Delete 70
Delete 52
Insert 37 -9294
Delete 33
Reverse 77 97
Reverse 173 174
Query 147 149 2
Delete 43
Insert 168 4425
Insert 72 792
Insert 123 -6172
Query 139 173 23
Insert 78 3718
Delete 25
Query 132 134 2
Delete 88
Insert 13 8268
Query 58 164 14
Query 57 118 16
Query 80 176 6
Insert 49 -1553
Reverse 166 166
Query 134 155 21
Insert 134 2562
Insert 133 6009
Query 149 159 7
Delete 56
Insert 100 4886
Delete 50
Insert 46 -3457
Query 18 35 5
Reverse 167 176
Reverse 95 95
Reverse 120 179
Insert 4 -5294
Reverse 104 117
