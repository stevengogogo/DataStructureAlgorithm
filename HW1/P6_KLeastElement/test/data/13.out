-3160
-8275
-9273
-1487
-9731
-3846
1728
-748
9325
-2818
-8503
-8624
499
-1047
2458
-6818
9340
-4893
8964
4426
-4626
-2098
7858
8470
-4445
-5333
-2986
3926
9518
1930
-2712
-493
-2445
-7161
9894
4057
46
-3496
933
-8654
-1640
2458
-5542
7801
7377
8767
-5112
-1135
-3704
933
8236
-7197
-2581
-1680
165
7354
8649
-3812
165
-1679
5536
-3230
4057
-9363
2262
-9020
-3230
-3230
5227
6623
46
-8806
5258
-7501
-4133
-9166
-748
-5916
-9104
-1463
3926
8148
499
-6818
-4626
1930
9211
8715
-7379
7246
-7501
499
-5916
-4613
-7197
1096
8236
-7357
-5916
-7999
-5916
-1238
-7886
-8506
3107
9604
-9020
-7237
-3230
325
4057
9518
-8759
3926
7827
-3846
-3704
7531
4980
1340
-5459
-7430
-5916
-8624
7541
888
-6786
